gulp/tasks.js

```diff
@@ -0,0 +1,41 @@
+/*
+  tasks.js
+  ===========
+  defaults wraps generate-assets, watch and server
+*/
+
+var gulp = require('gulp')
+var gutil = require('gulp-util')
+var runSequence = require('run-sequence')
+
+gulp.task('default', function (done) {
+  runSequence('generate-assets',
+                'watch',
+                'server', done)
+})
+
+gulp.task('generate-assets', function (done) {
+  runSequence('clean',
+                'copy-govuk-modules',
+                'sass',
+                'sass-documentation',
+                'copy-assets',
+                'copy-documentation-assets', done)
+})
+
+gulp.task('copy-govuk-modules', function (done) {
+  runSequence(['copy-toolkit',
+                'copy-template-assets',
+                'copy-elements-sass',
+                'copy-template'], done)
```

Comment:
Do these all actually need to run sequentially, or can they run concurrently? I think this could be:

```javascript
gulp.task('copy-govuk-modules', ['copy-toolkit', 'copy-template-assets', 'copy-elements-sass', 'copy-template'])
```

This would run a lot faster since it would spin up parallel threads for all the separate copying tasks. You can apply this to the other tasks in this file too; I think that with task dependencies (the optional array argument I used above) you shouldn't, in theory, need to ever use `runSequence`, though sometimes it can mean you have to do:

```javascript
gulp.task('the-first-thing', foo)
gulp.task('a-thing-before-others', ['the-first-thing'], bar)
gulp.task('earlierer-thing', ['a-thing-before-others'], baz)
gulp.task('earlier-thing', ['earlierer-thing'], qux)
gulp.task('thing', ['earlier-thing'], blah)
```

Which is an edge case that is obviously better suited to `runSequence`.